examples/pickup_vrp/loading_example.py

```diff
@@ -230,9 +218,10 @@ def check_solution(res, gpdp: GPDP):
         n_solutions=10000,
     )
     logging.basicConfig(level=logging.DEBUG)
-    results = solver.solve_intern()
```

Comment:
I think there are other functions in this file that are using the old api `solve_intern()`. As they are not called in `if __name__ == "__main__": `block, the example is still working though. I made the modifications in my branch on ortools solver, so maybe you can let it as is and i will push my own modifications later.

Reply:
done